Formula/c/cucumber-ruby.rb

```diff
@@ -26,72 +26,72 @@ class CucumberRuby < Formula
   uses_from_macos "libffi", since: :catalina
 
   # Runtime dependencies of cucumber
-  # List with `gem install --explain cucumber`
+  # List with `gem install --explain cucumber | grep -v 'Gems to install' | sort`
```

Comment:
it would be better not to sort (sometimes, it would cause issue)

Reply:
I just removed this shell example but I left the gem resources sorted. If it's preferable to not sort the resources either I can revert that too.

Reply:
Sometimes the dep order matters. One gem may require another one is already installed before it can be. Better to switch the order back and just bump the necessary versions.